future-apps/payroll/app/src/panels/AddEmployee.js

```diff
@@ -6,10 +6,10 @@ import { Button, Field, SidePanel } from '@aragon/ui'
 import { startOfDay } from 'date-fns'
 
 import Input from '../components/Input'
-import AragonContext from '../context/AragonContext'
+import AragonContext, { connect } from '../context/AragonContext'
```

Comment:
I think we can remove `AragonContext` if we are using `connect`, here and in ln 29

Reply:
We must not remove AragonContext, without it we loose app context.

https://reactjs.org/docs/context.html#classcontexttype

Reply:
done!